backend/docker-entrypoint.sh

```diff
@@ -14,10 +14,19 @@ if [[ "$APPLY_MIGRATIONS" = "1" ]]; then
     ./manage.py migrate --noinput
 fi
 
-# Create superuser
-if [[ "$CREATE_SUPERUSER" = "1" ]]; then
-    ./manage.py add_admin_user -u admin -p admin -e admin@example.com
-    echo "Admin user created with credentials admin:admin (email: admin@example.com)"
+# Create admin user. Generate password if there isn't one in the
+# environment variables. Password is not printed to log, but needs to be
+# changed at the pod command lineif [[ "$CREATE_SUPERUSER" = "1" ]]; then
```

Comment:
There's something extra in this comment.

Reply:
Whoops :D